server/app/views/JsBundle.java

```diff
@@ -27,6 +27,6 @@ public enum JsBundle {
   }
 
   public String getJsPath() {
-    return jsPath;
+    return "dist/" + jsPath;
```

Comment:
We discussed and we need to add the `dist` here so that the admin and applicant bundle paths are correct. They previously were prepended by `javascripts` in `ViewUtils.java`